node-tests/unit/utils/config-reader-test.js

```diff
@@ -26,4 +26,18 @@ describe('ConfigReader | readTestemConfig', function() {
       'bar'
     );
   });
-});
+
+  it('should read a specified `json` file and return an object read from the file', function() {
+    assert.deepEqual(
+      readTestemConfig('testem.json-file.json', []).foo,
```

Comment:
Suggestion: all the mock testem config file that are used in this test module.. lets use fixturify to write these files instead of having it in the root of the repo.
```
const cwd = path.resolve('/');  // not sure about this?
  describe('ConfigReader ...', function() {
    beforeEach(function() {
      fixturify.writeSync(cwd, {
        'mock-testem.json': JSON.stringify({
           "foo": "bar"
        })
      });
    });
```